op-e2e/system_test.go

```diff
@@ -1522,15 +1549,17 @@ func TestChallenge(t *testing.T) {
 				require.Equal(t, output.Submitter, cfg.Secrets.Addresses().Challenger1)
 				outputDeleted := val.IsOutputDeleted(output.OutputRoot)
 				require.True(t, outputDeleted)
```

Comment:
I suppose it's a matter of taste, but I think the `val.IsOutputDeleted` function is so clear that you don't need to do local variable assignments.

```go
require.True(t, val.IsOutputDeleted(output.OutputRoot))
```